hack/ginkgo-e2e.sh

```diff
@@ -218,6 +218,22 @@ suite_args+=(--report-complete-ginkgo --report-complete-junit)
 # environments like Prow.
 case "${GINKGO_SHOW_COMMAND:-${CI:-no}}" in y|yes|true) set -x ;; esac
 
+# The e2e.test binary is meant to be self-contained and must embed all
+# files that it depends on. For example, it gets run outside of the
+# Kubernetes source tree during conformance testing. The mechanism
+# for loading files from the current directory (required for
+# user-configurable files e.g. when running the storage tests against
+# an out-of-tree CSI driver) can hide missing embedded files.
+#
+# To detect missing embedding, we change the directory here when running
+# in a CI environment. Otherwise we do the opposite and explicitly
```

Comment:
I'm a bit lost now about why do we need the `--repo-root`, was it only used for the files with the fixtures?

Reply:
I preserved it because it was there to begin with and I didn't want to break someone who relied on it.

I think it is safe to remove it, but that then extends the scope of this PR.